### PR TITLE
feat: Initial design of the manage submissions page.

### PR DESCRIPTION
**Pull request by Sruthin86**
Review Checklist
 - [x] Create a new view for the Submissions page.
 - [x] Add a h3 for the dropdown.
 - [x] Add and h3 for reviewers section
 - [x] Create a dropdown to demonstrate the addition of reviewers to a submission.
 - [x] Create a list to display the added reviewers.
 - [x] Add a user avatar in the list.
 - [x] Add email address under the display name.

Closes Issue #400

### CODE REVIEW

File: client/src/pages/SubmissionDetails.vue [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+          <h3>Add a Reviewer</h3>
+          <div class="q-gutter-md row">
+            <q-select
+              v-model="model"
+              filled
+              use-input
+              hide-selected
+              fill-input
+              input-debounce="0"
+              hint="Minimum 2 characters to trigger filtering"
```
Comment (gmeben) [inline client/src/pages/SubmissionDetails.vue:18]: Screen Reader for Chrome (ChromeVox) only announces the hint when this element is tabbed, which means Screen Reader should read something that more clearly describes its purpose.
```suggestion
              hint="Search for a user to assign."
```

**Review by gmeben — CHANGES_REQUESTED** [review]

**Comment by gmeben** [comment]
It occurred to me that this is missing a Jest and Cypress test.

**Review by Sruthin86 — COMMENTED** [review]
Grant, I looked at your changes and approved them. Do you have to approve the review so that this can be merged into development and master?

**Comment by gmeben** [comment]
@Sruthin86 Yes, it has to be approved by another user before it's eligible to be merged. I didn't approve it because the Cypress and Jest tests were still missing. I've added them, so now I can go through your review checklist.

**Pull request merged**